### PR TITLE
Add Gonzalo as Cpt for 3.25.2 release

### PR DESCRIPTION
**Pull request by asdine**
Prepare for https://github.com/sourcegraph/sourcegraph/issues/18693

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ All notable changes to Sourcegraph are documented in this file.
 
 ## 3.25.1
 
+### Fixed
+
+- A security vulnerability with in the authentication workflow has been fixed. [#18686](https://github.com/sourcegraph/sourcegraph/pull/18686)
```
Comment (davejrt) [inline CHANGELOG.md:37]: This should be under 3.25.2 correct?

**Review by davejrt — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pecigonzalo) [reply CHANGELOG.md:37]: :fail: yes!

Comment (pecigonzalo) [reply CHANGELOG.md:37]: @davejrt fixed, thanks for the spot!

**Review by davejrt — APPROVED** [review]
See commect about the dates

File: dev/release/release-config.jsonc [hunk 2]
```diff
@@ -10,16 +10,16 @@
...
     // Release versions
     "previousRelease": "3.25.1",
-    "upcomingRelease": "3.26.0",
+    "upcomingRelease": "3.25.2",
 
     // Release dates
-    "releaseDate": "20 March 2021 10:00 PST",
-    "oneWorkingDayAfterRelease": "22 March 2021 10:00 PST",
+    "releaseDate": "26 March 2021 10:00 PST",
+    "oneWorkingDayAfterRelease": "26 March 2021 10:00 PST",
```
[the remark this answers is not in the document]
Comment (pecigonzalo) [reply dev/release/release-config.jsonc]: 🤦 fixed!

**Review by davejrt — APPROVED** [review]
Cool, looks better now :-)

**Pull request merged**